### PR TITLE
feat: add acute toxicity ld50 from TDC

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

File: data/Acute_Toxicity_LD50/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - conservative dose that can lead to lethal adverse effects.
+  - Rat Acute Toxicity by Oral Exposure
+  - Toxicity
+identifiers:
+- id: SMILES
+  type: SMILES
+  description: SMILES
+- id: compound_name
+  type: Other
+  description: compound name
```
Comment (kjappelbaum) [inline data/Acute_Toxicity_LD50/meta.yaml:21]: is this a IUPAC-like name?

File: data/Acute_Toxicity_LD50/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- id: acute_toxicity
+  description: Acute Toxicity LD50.
+  units: ld50
+  type: continuous
+  names:
+  - Acute Toxicity LD50
+  - ld50
+  - conservative dose that can lead to lethal adverse effects.
+  - Rat Acute Toxicity by Oral Exposure
+  - Toxicity
```
Comment (kjappelbaum) [inline data/Acute_Toxicity_LD50/meta.yaml:14]: @MicPie I'm really thinking more and more that our model might benefit from also having descriptions about what those things mean. We should decide of how we approach this and what we aim for as "names" in the prompts. I'm hesitant with general things such as "toxcitiy" as they can mean too many things and we now have the chance to really create a high-quality dataset

File: data/Acute_Toxicity_LD50/meta.yaml (hunk 1 above)
Comment (phalem) [reply data/Acute_Toxicity_LD50/meta.yaml:21]: @kjappelbaum May be I made a mistake. I will try to edit all PR again and remove compound id as well.

Comment (phalem) [reply data/Acute_Toxicity_LD50/meta.yaml:21]: yeah, Synonyms. some follow iupac name and some follow other names. I change it to Synonyms as I found most compound in PubChem Synonyms.

**Pull request merged**